Local and Remote Options were added to the IOC

danielmaclaren opened the pull request:
- https://github.com/ISISComputingGroup/IBEX/issues/7878

danielmaclaren committed:
Local and Remote Options were added to the IOC

FreddieAkeroyd commented on aeroflexSup/aeroflex_common.db:
```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
     field(ONAM, "COMMS DISABLED")
 }
 
+record(bi, "$(P)CMODE_RBV") {
```
AS per https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming#signal-qualifiers this should be just `CMODE` with optionally an alias to `CMODE:SP:RBV`, this naming convention helps setting blocks work from python as you set a block to "CMODE" and it adds `:SP` if you write from python

esmith1729 replied:
Changed PV name to "CMODE", and added alias for SP:RBV as well

FreddieAkeroyd commented on aeroflexSup/aeroflex_common.db:
```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
+	field(ZRVL, "0")
+	field(ONVL, "1")  
+    field(OUT, "$(P)DISABLE PP")
+    field(FLNK, "$(P)CMODE_RBV.PROC")
```
FLNK not needed as CMODE is CP

esmith1729 replied:
Removed FLINK

FreddieAkeroyd commented on aeroflexSup/aeroflex_common.db:
```diff
@@ -17,6 +17,27 @@ record(bo, "$(P)DISABLE")
+    field(ZRST, "REMOTE")
+    field(ONST, "LOCAL")
+	field(ZRVL, "0")
+	field(ONVL, "1")  
```
Does this load OK? I think bo used ZNAM/ONAM instead

esmith1729 replied:
Changed to ZNAM and ONAM.

esmith1729 committed:
Update aeroflex_common.db with req'd changes

- Changed CMODE_RBV to "CMODE" with alias of "CMODE:SP:RBV"
- Removed FLINK from line 38, record CMODE:SP
- Change ZVRL and ONVL to ZNAM and ONAM

esmith1729 committed:
Removed alias field causing issues

FreddieAkeroyd committed:
Fix ZRST -> ZNAM etc in bo record
